### DISCUSSION TITLE
Generelle Überarbeitung der Zustände

### DISCUSSION THREAD

**Discussion by Gorthian** [Gorthian/HeXXen1733_Custom_Character_Sheet#27] [Ideas]
Stand heute sind im Regelwiki 30+ Zustände gelistet.
Kann man das irgendwie sinnvoll im Sheet darstellen? 
Oder sollte auf Zustände im Sheet komplett verzichtet werden?

**Comment by opatut** [comment]
Ich finde die Tabelle unglaublich nützlich, um das doch etwas komplizierte indirekte System (Zustand vs. Einfluss) zu durchschauen und vor allem nicht ständig nach den Abbau-Regeln schauen zu müssen.

Vielleicht funktioniert eine editierbare Liste zum Selbsteinpflegen, wie die Allgemeinen Fertigkeiten/Espritkräfte? Das wäre wohl das flexibelste, vielleicht geht das ja auch mit "Defaults" (z.B. die Zustände aus dem Hauptregelwerk) die sich dann von Hand erweitern lassen?

**Reply by Gorthian** [reply]
Am liebsten wäre mir ja eine DropDown-Liste mit den Zuständen über die man die Zustände einblenden kann, an denen der Jäger aktuell "leidet".
Ich habe nur noch keine Idee ob und wie sich das in Roll20 umsetzen lässt.

**Comment by Gorthian** [comment]
Einen "Proof-Of-Concept" hätte ich mit 79f07eeea31f33c6df8a4e089521d5b35bb583e4 im Branch Zustände.
Sprich: Ich könnte Zustände über einen Drop-Down der Liste hinzufügen.
Das wäre dann "nur noch" Fleißarbeit alle Zustände in das Skript einzupflegen.

Wegfallen würde dadurch die Möglichkeit direkt auf die Proben zum Abbauen der Zustände zu würfeln.
Dafür habe ich in der Variante noch keine Lösung.

**Comment by Gorthian** [comment]
Mit Version 2.1 habe ich die Darstellung nochmal komplett überarbeitet.
Man kann sich jetzt über eine Drop-Down-Liste aus den Standard-Zuständen des Regelwikis (Stand heute) eine eigene Liste erstellen, udn zusätzlich weitere eigene Zustände definieren.
